operate/quickstart/run_service.py

```diff
@@ -70,7 +70,6 @@
     "name": "No staking",
     "description": "Your Olas Predict agent will still actively participate in prediction\
         markets, but it will not be staked within any staking program.",
-    "available_staking_slots": "∞",
```

Comment:
You could have simplified this PR as 

    "available_staking_slots": "n/a",
    
    (n/a not applicable)

Reply:
yeah, but I went with this approach to not overload the user with useless info